ray-operator/config/manager/kustomization.yaml

```diff
@@ -1,6 +1,8 @@
 images:
- - name: controller
-   newTag: latest
-   newName: ray-controller
+- name: controller
+  newName: seedjeffwan/ray-operator
```

Comment:
we should remove the repo: `seedjeffwan` from the name

Reply:
Yeah, I will remove it. This was used for e2e testing.